doc/test-cases.md

```diff
@@ -0,0 +1,36 @@
+# Test Cases
+## Purpose
+Goal of this document is to document non trivial testing scenarious we can use to ensure network module is not seriously broken.
+
+## Test Cases
+
+### Complex setup
+
+Test case require VM with two network cards. Setup is:
+
```

Comment:
I would start even with the more simplest ones. 

1. DHCP configuration (no configuration at all but different backends)
2. Static configuration ( 1 network card is enough, that will need some routing and DNS configuration by sure)
...
And then the complex ones.

A Goal for the future is to have the same network configuration done during the installation in the final machine independently of the selected backend (**not for now**)